deploy/vm/fetch-test-dicom.sh

```
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
+# Fetch DICOM studies from a source Orthanc, anonymize, and save locally.
+# Used by vm.sh cmd_bake() to prepare test data for the golden image.
+#
+# Usage: fetch-test-dicom.sh [--url URL] [--patient NAME] [--limit N] [--output DIR]
+#   Defaults come from vm.conf (DICOM_SOURCE_URL, DICOM_SOURCE_PATIENT).
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+DEPLOY_DIR="$(cd "$SCRIPT_DIR/.." && pwd)"
+
+source "$DEPLOY_DIR/lib/logging.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./lib/logging.sh was not specified as input (see shellcheck -x).

(SC1091)